src/Microsoft.OData.Core/Microsoft.OData.Core.csproj

```diff
@@ -77,20 +77,6 @@
     </Content>
   </ItemGroup>
 
-  <ItemGroup>

```

Comment:
why do we remove them? They are needed for the resource generation

Reply:
Ping @corranrogue9

Reply:
I recall having an intermediate commit in my PR where I had accidentally removed this section, but these changes are not in the master branch, and I agree with Sam that they should stay.

Reply:
@corranrogue9  This [commit](https://github.com/OData/odata.net/commit/bae11a51b2e0be642e2e42351774f14508becf55) is in the master branch and it did some changes to csproj. Kindly have a look